app/assets/stylesheets/_variables.scss

```diff
@@ -18,6 +12,7 @@ $header-line-height: 1.25;
 // Other Sizes
 $base-border-radius: 3px;
 $base-spacing: $base-line-height * 1em;
+$small-spacing: $base-spacing / 2;
```

Comment:
I added small spacing because it is used frequently in Bitters. I don't think we have a need for `$large-spacing` so I prefer not to include it.